apps/extension/src/routes/popup/home/index.tsx

```diff
@@ -61,13 +62,11 @@ export const PopupIndex = () => {
 
         <div className='grow' />
 
-        {!!frontendUrl && (
-          <a href={frontendUrl} target='_blank' rel='noreferrer'>
-            <Button className='flex w-full items-center gap-2' variant='gradient'>
-              Manage portfolio <ExternalLink size={16} />
-            </Button>
-          </a>
-        )}
+        <a href={frontendUrl} target='_blank' rel='noreferrer'>
+          <Button className='flex w-full items-center gap-2' variant='gradient'>
+            Manage portfolio <ExternalLink size={16} />
+          </Button>
+        </a>
```

Comment:
For existing wallets, `frontendUrl` may not be set. In this case click on this button does not lead to anything

Reply:
Good catch, thanks! I added a redirect to the settings page on this button in case the frontend URL is not set